source/numerics.tex

```diff
@@ -7279,40 +7250,42 @@
 \indexlibrarymember{operator\shl=}{valarray}%
 \indexlibrarymember{operator\shr=}{valarray}%
 \begin{itemdecl}
-valarray& operator*= (const valarray&);
-valarray& operator/= (const valarray&);
-valarray& operator%= (const valarray&);
-valarray& operator+= (const valarray&);
-valarray& operator-= (const valarray&);
-valarray& operator^= (const valarray&);
-valarray& operator&= (const valarray&);
-valarray& operator|= (const valarray&);
-valarray& operator<<=(const valarray&);
-valarray& operator>>=(const valarray&);
+valarray& operator*= (const valarray& v);
+valarray& operator/= (const valarray& v);
+valarray& operator%= (const valarray& v);
+valarray& operator+= (const valarray& v);
+valarray& operator-= (const valarray& v);
+valarray& operator^= (const valarray& v);
+valarray& operator&= (const valarray& v);
+valarray& operator|= (const valarray& v);
+valarray& operator<<=(const valarray& v);
+valarray& operator>>=(const valarray& v);
 \end{itemdecl}
 
 \begin{itemdescr}
 \pnum
+\requires
+\tcode{size() == v.size()}.
 Each of these operators may only be instantiated for a type \tcode{T}
-to which the indicated operator can be applied.
-Each of these operators
-performs the indicated operation on each of its elements and the
-corresponding element of the argument array.
+to which the indicated operator can be applied. 
```

Comment:
I'd like to point out that this is pre-existing text; and I think LWG failed in distant history to produce a specification for std::complex that would allow T's other than built-in floating-point types. I'm reluctant to attempt to come up with a better specification here. (Although I admit the std::complex case was probably more about trigonometric functions thatn simple stuff.)